src/pages/communication-page/components/privacy-notice/privacy-notice.scss

```diff
@@ -0,0 +1,20 @@
+privacy-notice {
+  .container {
+    margin: 40px 32px 0px 32px;
+    background-color: #f3f3f3;
```

Comment:
this should probably be `gds-grey-4` or `light` from standard colours

Reply:
I can only go with the shade the design requested , will check with Nad when he's back in and working and change if needed

Reply:
If you can spot the difference between #f3f3f3 and #f4f4f4 I'll give you £100 ;)